Jenkinsfile

```diff
@@ -342,7 +363,13 @@ git push origin HEAD:master
  "docker run" parameters which:
  - make the Docker instance on the host available inside a container (socket and command)
  - set common Makefile values (cachebust, cache populated from images if available)
- - source in $GOPATH as current directory
+ - source in current directory
+ - GOPATH alongside it
+ - HOME above it
```

Comment:
I couldn't understand why we choose this restriction of `HOME` and `GOPATH` relative to the current directory.

Reply:
Jenkins is configured to invoke us with the current directory on a fast, local disk. I could have hard-coded `/mnt/workspace` here, but that is only true for the current Jenkins configuration, so I use the current directory instead to find that path.

`HOME` gets set to the parent for code which uses `$HOME/.cache` - we want that to be fast.

`GOPATH` is then redundant because it defaults to something in `$HOME` (don't remember what exactly it is). It gets explicitly merely to document what it is.

Reply:
Thanks for the clarification.